codegen/scala-gen/src/test/scala/com/akkaserverless/codegen/scalasdk/ValueEntityTestKitGeneratorSuite.scala

```diff
@@ -164,6 +164,66 @@ class ValueEntityTestKitGeneratorSuite extends munit.FunSuite {
     assertNoDiff(sourceCode, expected)
   }
 
+  test("it can generate an specific integration test stub for the entity") {
+    val entity = createShoppingCartEntity()
+    val service = createShoppingCartService(entity)
+    val main = FullyQualifiedName("Main", packageNaming.copy(protoPackage = "com.example.shoppingcart"))
+
+    assertEquals(
+      ValueEntityTestKitGenerator.integrationTest(main, entity, service).content,
+      """package com.example.shoppingcart.api
+
+import com.akkaserverless.scalasdk.testkit.AkkaServerlessTestkit
+import com.akkaserverless.scalasdk.testkit.ValueEntityResult
+import com.akkaserverless.scalasdk.valueentity.ValueEntity
+import com.example.shoppingcart.Main
+import com.example.shoppingcart.domain
+import com.google.protobuf.Empty
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.time.Millis
+import org.scalatest.time.Seconds
+import org.scalatest.time.Span
+import org.scalatest.wordspec.AnyWordSpec
+
+// This class was initially generated based on the .proto definition by Akka Serverless tooling.
+//
+// As long as this file exists it will not be overwritten: you can maintain it yourself,
+// or delete it so it is regenerated as needed.
+
+class ShoppingCartServiceIntegrationSpec
+    extends AnyWordSpec
+    with Matchers
+    with BeforeAndAfterAll
+    with ScalaFutures {
+
+  implicit val patience: PatienceConfig =
+    PatienceConfig(Span(5, Seconds), Span(500, Millis))
+
+  val testkit = AkkaServerlessTestkit(Main.createAkkaServerless())
+  testkit.start()
+  implicit val system = testkit.system
```

Comment:
`system: ActorSystem` to avoid warnings in Intellij

Reply:
is that needed for running streaming calls, or something else? wonder if we should not include it since it exposes a bit too much low level Akka? btw, if it's for streams we should have it as Materializer since that is what we have in the contexts

Reply:
It's used for creating the Akka gRPC client

Reply:
ah, we could have a grpcClient utility method similar to what we provide in the context. That would become more unified. Can be a separate ticket.

Reply:
I created issue https://github.com/lightbend/akkaserverless-java-sdk/issues/568 for this idea